tests/test_ote_training.py

```diff
@@ -27,6 +27,7 @@
 from mmdet.apis.ote.extension.datasets.mmdataset import MMDatasetAdapter
 
 logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
What does it do?

Reply:
Without it logger has the default log level `ERROR` or `WARNING` and does not print info.

Reply:
Doesn't running `pytest --log-cli-level=INFO ...` basically do the same thing?

Reply:
I checked: yes, it makes the same (or similar) thing.
But I would like to turn on INFO log messages from this test file itself. Just to always have INFO messages from this file.
Note that I do not add a lot of them -- only 12 `logger.info` messages are added in this file.

Reply:
Ok. Removed.

Reply:
Pavel, since this is a minor change, I think we should not re-run the CI.
Do you agree with it?

Reply:
Agree!